python/ark/module.py

```diff
@@ -191,3 +202,74 @@ def forward(self, *args: Any, **kwargs: Any) -> Any:
 
             rt.run()
             return _recursive_ark_to_torch(self.forward_output)
+
+
+class ModuleFn(Function):
```

Comment:
Please rename this class to something starting with an underscore, as this isn't supposed to be called by users.